PWGHF/TableProducer/HFBPlusToD0PiCandidateSelector.cxx

```diff
@@ -151,7 +163,24 @@ struct HfBplusTod0piCandidateSelector {
         continue;
       }
 
-      hfSelBPlusToD0PiCandidate(1);
+      auto pidPion = -1;
```

Comment:
I don't see `pidPion` used anywhere.

Reply:
Yes, you're right. I'm removing it since is not useful for the selection